### PR TITLE
Update pre-release-week.md

### PR DESCRIPTION
**Pull request by curquiza**

### CODE REVIEW

File: guides/pre-release-week.md [hunk 1]
```diff
@@ -28,19 +28,18 @@ If a change involves a huge implementation in the integration tools, a separated
...
+The script automates the following tasks:
 
 - In each integration repository, create a draft PR modifying the old version of MeiliSearch into the future release version. The branch name originating this PR should be `bump-meilisearch-v*.*.*`. In most cases, only the `README.md` file changes.
-
-- Create a draft PR in the [meilisearch-digitalocean](https://github.com/meilisearch/meilisearch-digitalocean) repository modifying the old MeiliSearch version **into the RC version**. This branch originating the PR should be named `bump-meilisearch-v*.*.*-rc`. This branch is only created for test purposes and will be closed at the end of the pre-release week.
+- In the [cloud-scripts](https://github.com/meilisearch/cloud-scripts) repository, create a branch named `bump-meilisearch-v*.*.*-rc` where the old version of MeiliSearch is changed **into the RC version**. Also, create a tag named `v*.*.*-rc` on this commit and push it.
```
Comment (eskombro) [inline guides/pre-release-week.md:36]:
```suggestion
- In the [cloud-scripts](https://github.com/meilisearch/cloud-scripts) repository, create a branch named `bump-meilisearch-vX.X.X-rc` where the old version of MeiliSearch is changed **into the RC version**. Also, create a tag named `vX.X.X-rc` on this commit and push it.
```

Comment (eskombro) [reply guides/pre-release-week.md:36]: Aren't we using this syntax just for consistency with other repos? :)

File: guides/pre-release-week.md [hunk 2]
```diff
@@ -28,19 +28,18 @@ If a change involves a huge implementation in the integration tools, a separated
...
 
 ## 🧪 Testing
 
 - **Test manually the RC** with a Core team member or on your own. Should be done by all the Integration team members.
 - **Run all the SDKs automatic test suites against the RC** thanks to [this script](https://github.com/meilisearch/integration-scripts/tree/main/sdks-tests). If tests are not green, these should be justified.
-- **Test manually the DO image with the RC** on the `bump-meilisearch-v*.*.*-test` branch by following the [testing process steps](https://github.com/meilisearch/meilisearch-digitalocean/blob/master/CONTRIBUTING.md#test-before-releasing) **without merging the branch or submitting the image.**
+- **Test manually the DO image with the RC** on the `bump-meilisearch-v*.*.*-rc` branch by following the [testing process steps](https://github.com/meilisearch/meilisearch-digitalocean/blob/master/CONTRIBUTING.md#test-before-releasing) **without merging the branch or submitting the image.**
```
Comment (eskombro) [inline guides/pre-release-week.md:42]:
```suggestion
- **Test manually the DO image with the RC** on the `bump-meilisearch-vX.X.X-rc` branch by following the [testing process steps](https://github.com/meilisearch/meilisearch-digitalocean/blob/master/CONTRIBUTING.md#test-before-releasing) **without merging the branch or submitting the image.**
```

File: guides/pre-release-week.md [hunk 3]
```diff
@@ -56,7 +55,7 @@ Some tests still might fail on this main PR until the new release of MeiliSearch
 
 ## 🥳 After the MeiliSearch official release
 
-- Remove the `bump-meilisearch-v*.*.*-rc` branch from [meilisearch-digitalocean](https://github.com/meilisearch/meilisearch-digitalocean/branches).
+- Remove the `bump-meilisearch-v*.*.*-rc` branch and the `v*.*.*-rc` tag from the [cloud-scripts](https://github.com/meilisearch/cloud-scripts) repository.
```
Comment (eskombro) [inline guides/pre-release-week.md:58]:
```suggestion
- Remove the `bump-meilisearch-vX.X.X-rc` branch and the `vX-X-X-rc` tag from the [cloud-scripts](https://github.com/meilisearch/cloud-scripts) repository.

```bash
# Delete local branch
$ git branch -d bump-meilisearch-vX.X.X-rc

# Delete remote branch
$ git push --delete origin bump-meilisearch-vX.X.X-rc

# Delete tag
$ git push --delete origin v.X.X.X-rc
```

```

**Review by eskombro — DISMISSED** [review]
🌮  For @curquiza

File: guides/pre-release-week.md (hunk 3 above)
Comment (curquiza) [reply guides/pre-release-week.md]: I would rather not add these explanations. This makes things complex and all these changes can be done via the GitHub interface.

**Review by eskombro — APPROVED** [review]
🆗 👌🏽

**Pull request merged**